### PR TITLE
Minor clarifications to leftNav of Getting Started

### PR DESCRIPTION
**Pull request by williamkbentley**
- Changed dashboard links to "My iModels" and "My Applications"
- Added link to Tutorials

### CODE REVIEW

File: docs/getting-started/leftNav.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-##### [iModel Registration Dashboard](/getting-started/registration-dashboard?tab=1)
+##### [My iModels](/getting-started/registration-dashboard?tab=1)
 
-##### [Application Registration Dashboard](/getting-started/registration-dashboard?tab=0)
+##### [My Applications](/getting-started/registration-dashboard?tab=0)
 
 ---
 
-## [Offline](./offline-quickstart.md)
+## [Offline (Quick Start)](./offline-quickstart.md)
```
Comment (calebmshafer) [inline docs/getting-started/leftNav.md:9]: At some point should we link these to the Tutorials instead of these separate files?  That can be a different PR, just curious if we had plans to do it.

**Review by calebmshafer — APPROVED** [review]

File: docs/getting-started/leftNav.md (hunk 1 above)
Comment (williamkbentley) [reply docs/getting-started/leftNav.md:9]: Makes sense or Online; there is quite a but more info in develop-desktop-viewer.md. But I'm not sure it makes sense for Offline. "Quick Start" and develop-desktop-viewer are essentially the same thing.

**Comment by calebmshafer** [comment]
@Mergifyio backport release/2.12.x
